libpoolprotocols/stratum/EthStratumClient.cpp

```diff
@@ -573,9 +572,7 @@ void EthStratumClient::connect_handler(const boost::system::error_code& ec)
         jReq["method"] = "eth_submitLogin";
         if (m_worker.length())
             jReq["worker"] = m_worker;
-        jReq["params"].append(m_user + m_conn->Path());
-        if (!m_email.empty())
-            jReq["params"].append(m_email);
+        jReq["params"].append(m_user + "/" + m_conn->Path());
```

Comment:
We should do a conditional for empty on m_conn->Path() otherwise we always end up with an user and a trailing slash

Reply:
Indeed.. let me fix